data/src/main/kotlin/com/nexters/bandalart/data/local/datasource/GuestLoginTokenLocalDataSource.kt

```diff
@@ -0,0 +1,37 @@
+package com.nexters.bandalart.data.local.datasource
+
+import androidx.datastore.core.DataStore
+import androidx.datastore.preferences.core.Preferences
+import androidx.datastore.preferences.core.edit
+import androidx.datastore.preferences.core.emptyPreferences
+import androidx.datastore.preferences.core.stringPreferencesKey
+import com.nexters.bandalart.data.datasource.GuestLoginTokenDataSource
+import com.nexters.bandalart.data.local.datastore.DataStoreKeys
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.catch
+import kotlinx.coroutines.flow.map
+import java.io.IOException
+
+class GuestLoginTokenLocalDataSource(private val dataStore: DataStore<Preferences>) : GuestLoginTokenDataSource {
+
+  private val key = stringPreferencesKey(DataStoreKeys.GUEST_LOGIN_TOKEN)
+
+  override suspend fun set(guestLoginToken: String) {
+    dataStore.edit { preferences ->
+      preferences[key] = guestLoginToken
+    }
+  }
+
+  override suspend fun get(): Flow<Result<String>> {
```

Comment:
```suggestion
override fun get(): Flow<Result<String>> 
```
get 하는 함수는 반환타입이 flow 이기 때문에 suspend keyword 가 없어도 됨 !